### PR TITLE
Flesh out D7 integration documentation

### PR DESCRIPTION
**Pull request by derekderaps**

### CODE REVIEW

File: docs/5-Integrations.md [hunk 1]
```diff
@@ -12,18 +12,47 @@
...
-- When sending other theme output through twigshim_render() (e.g., other Paragraph bundles, blocks, node templates), you must set up the Twig template variables manually and pass them in to twigshim.
+
+[Twigshim](https://github.com/kalamuna/twigshim) is a small helper 
+[module](https://www.drupal.org/docs/7/extending-drupal-7/installing-contributed-modules) 
+for [Drupal 7](https://www.drupal.org/drupal-7.0) that pulls in
+[Symfony](https://symfony.com/)'s [Twig](http://twig.sensiolabs.org/) library 
+via [Composer](https://getcomposer.org/). It is an "API module", which means it 
+doesn't do anything on its own; it just exposes a `twigshim_render()` function 
+that accepts a Twig template file path and an array of template variables.
+Developers can use this function to bypass Drupal's standard render pipeline and
```
Comment (RobLoach) [inline docs/5-Integrations.md]: Splitting line breaks at 60/80 characters is a pain to maintain for documentation. While we could leave it for this, I request we avoid that going forwards.

File: docs/5-Integrations.md [hunk 2]
```diff
@@ -12,18 +12,47 @@
...
+via [Composer](https://getcomposer.org/). It is an "API module", which means it 
+doesn't do anything on its own; it just exposes a `twigshim_render()` function 
+that accepts a Twig template file path and an array of template variables.
+Developers can use this function to bypass Drupal's standard render pipeline and 
+leverage Twig templates instead. There are various ways you can hook into the 
+render pipeline and override it. The most common way is to define or override a 
+[theme 
+implementation](https://api.drupal.org/api/drupal/includes%21theme.inc/function/theme/7.x)
+at the level of nodes, fields, blocks, or anything else 
+([Paragraph](https://www.drupal.org/project/paragraphs) entities are an
```
Comment (soniktrooth) [inline docs/5-Integrations.md]: I don't think the parenthesis are needed here, just make it a new sentence.

**Comment by derekderaps** [comment]
Note to self, some code sample here to use in the docs: [gist](https://gist.github.com/derekderaps/a98c23b1ab444cd4fc5ba377d63ed8d3)

File: docs/5-Integrations.md (hunk 1 above)
Comment (derekderaps) [inline docs/5-Integrations.md]: > this isn't code

# LIFE IS CODE!!! 🔢

File: docs/5-Integrations.md [hunk 3]
```diff
@@ -12,18 +12,47 @@
 
 ## Drupal 7
 ### Using twig in D7
```
Comment (derekderaps) [inline docs/5-Integrations.md]: Your face should be capitalised

**Comment by derekderaps** [comment]
Okay @soniktrooth What do you think of it now?

**Pull request merged**